Components/Core/Conditions/Categories/Location.lua

```diff
@@ -172,8 +172,39 @@ ConditionCategory:RegisterCondition(1,	 "INSTANCE2", {
 	end,
 })
 
+ConditionCategory:RegisterCondition(1.1,  "MYTHICKEYSTONE", {
+    text = L["CONDITIONPANEL_KEYSTONE"],
+    tooltip = L["CONDITIONPANEL_KEYSTONE_DESC"],
+    min = 0,
+    max = 30,
+    unit= false,
+    icon = "Interface\\Icons\\inv_relics_hourglass",
+    tcoords = CNDT.COMMON.standardtcoords,
+
+    Env = {
+    	GetActiveKeystoneInfo = function()
+    		return select(1, C_ChallengeMode.GetActiveKeystoneInfo()) or 0
```

Comment:
Don't need to `select` the first return of a function - just return `C_ChallengeMode.GetActiveKeystoneInfo() or 0` directly. (I'm assuming that the signature is `local keystoneLevel, ... = C_ChallengeMode.GetActiveKeystoneInfo()`)